app/Exceptions/QuizNotVerifiedException.php

```diff
@@ -8,9 +8,9 @@ class QuizNotVerifiedException extends Exception
 {
     public function render($request)
     {
-        if(!$request->expectsJson()) {
+        if (!$request->expectsJson()) {
```

Comment:
```suggestion
        if (! $request->expectsJson()) {
```
Are we not aiming consistent linting here?

Reply:
This is a result of removing old style-ci configuration and using php-cs-fixer to perform style linting using GH actions.
- Added New Php-cs-fixer dev-dependency.
- Removed old configuration.
- Added new configuration without any custom rules, simply using existing @PSR12 ruleset.
- Ran the fixer once on all of codebase.